src/test/java/com/blog/config/SpringTestConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.blog.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
+
+public class SpringTestConfig {
```

Comment:
Think it's redundant because you're using `@Mock` annotation in tests, so there're no necessity to have Spring configuration

Reply:
This is fixed in the last commit.